packages/core/src/http-client/csrf-token-header.ts

```diff
@@ -58,7 +62,10 @@ function makeCsrfRequest<T extends HttpRequestConfig>(
   return executeHttpRequest(destination, appendSlash(axiosConfig))
     .then(response => response.headers)
     .catch(e1 => {
-      logger.error(
+      if (hasCsrfToken(e1)) {
+        return e1.response.headers;
+      }
+      logger.warn(
```

Comment:
I know the message was already there but coud you correct it to something like:
`'Initial try to fetch CSRF token failed - retry without slash at end : ${URL where the request goes to.}'`